packages/solid-crs-manage/lib/i8n/nl-NL.json

```diff
@@ -124,6 +124,21 @@
         "locale": "nl-NL",
         "value": "Collectie aangemaakt"
     },
+    {
+        "key": "nde.features.collections.root.delete.confirm",
+        "locale": "nl-NL",
+        "value": "Verwijderen"
+    },
+    {
+        "key": "nde.features.collections.root.delete.cancel",
+        "locale": "nl-NL",
+        "value": "Annuleren"
+    },
```

Comment:
These can probably go under the same key, under "common" at the bottom of the file. No need to change it in this PR, perhaps keep in mind when doing https://github.com/netwerk-digitaal-erfgoed/solid-crs/issues/336

Reply:
(same key as the object confirm and cancel)